src/__tests__/aggregatorExecuteLooksRareV2.test.ts

```diff
@@ -66,21 +66,37 @@ describe("LooksRareAggregator class", () => {
 
     const signature = await maker._signTypedData(domain, utils.makerTypes, makerOrder);
 
+    const referrerFromAPI: Referrer = {
+      rate: 0,
+      address: referrer?.address || constants.HashZero,
+    };
     // Fake an order from the API
-    const makerOrderFromAPI: MakerOrderFromAPI = {
+    return {
       merkleTree,
       signature,
+      referrer: referrer ? referrerFromAPI : undefined,
```

Comment:
so if referrer address is blank then there is no point defining `referrerFromAPI` even?

Reply:
True, fixed!